CHANGELOG.md

```diff
@@ -1,6 +1,166 @@
 kOS Mod Changelog
 =================
 
+# v1.0.0 (for KSP 1.1.3) Hey let's stop calling it Beta.
+
+### About the name:
+
+kOS has been around long enough that we figured it was long overdue
+for us to stop calling it 0.something.  Lots of people are using it,
+and we're worried about backward compatibility enough that we're not
+really treating it like a Beta anymore.  This version contains mostly
+a few things that we knew might break backward compatibility so we'd
+been putting them off for a long time.  A jump to 1.0 seems a good time
+to add those changes.
+
+Of course, it has lots of other changes for whatever else was being 
+worked on since the last release.
+
+### BREAKING CHANGES
+
+* As always, if you use the compiler feature to make KSM files, you should
+  recompile the KSM files when using a new release of kOS or results will
+  be unpredictable.
+* New Subdirectories ability has deprecated several filename commands such
+  as ``delete``, ``copy``, and ``rename``.  They will still work, but will
+  complain with a message every time you use them, as we may be removing
+  them eventually.  The new commands ``deletepath``, ``copypath``, and
+  ``movepath`` described below are meant to replace them.
+* When using a RemoteTech antenna that requires directional aiming,
+  in the past you could aim it at mission control with
+  ``SETFIELD("target", "Mission Control")`` and now you have to
```

Comment:
That should actually be reversed, it used to be `mission-control`, now it's `Mission Control`. That's because `mission-control` was kind of a reserved keyword before, now we support multiple ground stations so we have to reference them by name.